src/main/assets/scss/main.scss

```diff
@@ -1,10 +1,35 @@
 @import '~govuk-frontend';
 
+$line-colour: #1d70b8;
+
 .govuk-header__navigation--end {
   margin: -5vh 0 0;
   padding: 1vh 0 0;
 }
 
+.line-bottom {
```

Comment:
we should be using the gov classes and extending those if we need to, check out https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/ , theres loads there that arent on the examples on toolkit and this is the stylesheet we use

Reply:
I've updated the <hr> to use the Gov style class, and extended that instead